src/fairchem/core/trainers/ocp_trainer.py

```diff
@@ -291,6 +291,10 @@ def _forward(self, batch):
                     cg_change_mat(_max_rank, self.device),
                     pred_irreps,
                 )
+            else:
+                raise ValueError(
+                    f"Output target {target_key} not found in model output"
```

Comment:
let's also print out as well to make it super obvious from the error message
also nit: semantically this is an AttributeError

Reply:
ok, updated this